wled00/bus_manager.h

```diff
@@ -107,13 +107,15 @@ class BusDigital : public Bus {
   public:
   BusDigital(BusConfig &bc, uint8_t nr) : Bus(bc.type, bc.start) {
     if (!IS_DIGITAL(bc.type) || !bc.count) return;
-    _pins[0] = bc.pins[0];
-    if (!pinManager.allocatePin(_pins[0])) return;
+    uint8_t pin = bc.pins[0];
```

Comment:
Unnecessary. You can `allocatePin(bc.pins[0])` and if successful assign `_pins[0] = bc.pins[0];`

Reply:
True.  My style comes from programming in adverse environments, where buffers can (and are) changed beneath you, causing Time-of-Check/Time-of-Use bugs.

Doesn't apply here, so I've updated the code per your comments in bef9711.